Makefile

```diff
@@ -1,24 +1,27 @@
 PROJECT = rabbitmq_email
 PROJECT_DESCRIPTION = RabbitMQ plugin that converts incoming emails into messages and messages into outgoing emails
+RABBITMQ_VERSION = v3.9.x
 
-# use the patched gen_smtp from a fork
-dep_gen_smtp = git https://github.com/gotthardp/gen_smtp.git master
-dep_eiconv = git https://github.com/zotonic/eiconv.git master
+# Note: must be the same version as in rabbitmq-components.mk
+DEP_RANCH_VERSION = 2.1.0
 
-BUILD_DEPS += gen_smtp
+dep_amqp_client = git https://github.com/rabbitmq/rabbitmq-server.git $(RABBITMQ_VERSION)
+dep_rabbit_common = git https://github.com/rabbitmq/rabbitmq-server.git $(RABBITMQ_VERSION)
+dep_rabbit = git https://github.com/rabbitmq/rabbitmq-server.git $(RABBITMQ_VERSION)
+dep_rabbitmq_ct_client_helpers = git https://github.com/rabbitmq/rabbitmq-server.git $(RABBITMQ_VERSION)
+dep_rabbitmq_ct_helpers = git https://github.com/rabbitmq/rabbitmq-server.git $(RABBITMQ_VERSION)
```

Comment:
You need to use `git_rmq-subfolder` for those (same as in rabbitmq-components`, only the version should change).